scripts/clean-up-svg-icons.mjs

```diff
@@ -46,6 +46,7 @@ import { execSync } from 'child_process';
 // A list of icons that should not NOT be cleaned up. Their internal structure should not be changed
 // because, for instance, CSS animations may depend on it.
 const EXCLUDED_ICONS = [
+	'settings.svg',
```

Comment:
Is this required? Can we generate the icon so it is safe to be "cleaned up"?

Reply:
Nope, it don't work. Compressor breaks SVG icon.

![image](https://github.com/user-attachments/assets/cd672d78-4bd9-4f56-aa59-4218036a0ffe)